portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,60 @@
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Adam's Personal Portfolio</h1>
+      <p>This is my portfolio. I just finished sophmore year and this is my first time making a webpage.</p>
+
+      <p>I grew up in San Francisco where I went to a bilingual french school. My grandmother is french so it was 
+          very important to her and my mom that I was fluent in french as well. An interesting fact about me is 
+          that I am an Australian citizen since my father was born and raised in Sydney. In highschool I spent most of my free time playing
+          basketball and soccer, as well as playing the piano. Since going to college I've taken up rock climbing, which has unintentionally 
+          replaced soccer. I've met some of my closest friends through rock climbing and have really enjoyed our trips together when we 
+          go climbing outside. I have a twin sister who goes to Boston University, she's my favorite study and travel buddy. 
+      </p>
+
+      <p>I now attend the University of Pennsylvania where I'm studying Math and Computer Science. I had no computer science experience going into college and actually thought I would
+          double major in Math and Physics; I would take some CS courses to help with data analysis. But after my first programming class I realized that I was more interested in studying 
+          CS than physics.
+      </p>
+
+      <p>
+          Here are some of the projects I've worked on:
+        <ul>
+           <li>Cosmology research assistant (summer 2019): Responsible for splicing and calibrating data obtain from HERA telescope in order to image deep space</li>
+           <li>Sudoku Solver</li>
+           <li>Implement MineSweeper from scratch</li>
+           <li>Create a HashMap and TrieMap in Java</li>
+           <li>Code an auto-correct program</li>
+        </ul>
+      </p>
+
+      <p>Click <a href="https://github.com/adamfweidman"><button type="button">here</button></a> to for my Github account</p>
+      <p>Click <a href="https://linkedin.com/in/adam-weidman-0a42b217b/"><button type="button">here</button></a> to for my LinkedIn page</p>
+
+
+
+      <p>During these trying times I find myself doing two things most of the time: spending time with my dog 
+          and eating.<p>
+
+      <p>Here are some photos of my dog luna enjoying the sun:</p>
+      <img src="luna/luna-beach.jpg"/>
+      <img src="luna/luna-hat.JPG" width="450" height="550"/>
+
+      <p>Here are a couple pictures of food that I thought tasted particularly scrumptious: </p>
+      <img src="food/eggs.jpg" width="300" height="300"/>
+      <img src="food/pesto.JPG" width="250" height="300"/>
+
+      <p>Click <a href="random-pair.html"><button type="button">here</button></a> to see a random pairing of Luna and Food!</p>
```

Comment:
At Google, we like to keep the length of lines beneath 80 characters. It makes the lines far more readable (especially for reviewers!), since reading the code doesn't require using the scroll bar at the bottom of the page.

We won't be super pedantic about this, since at Google we have linters that remind us and also character markers in our IDE. As far as I know, Google Cloud Shell doesn't have this. So best effort on this :)